packages/perseus/src/types.ts

```diff
@@ -296,6 +296,11 @@ export type APIOptions = Readonly<{
      * only after a good few seconds.
      */
     editorChangeDelay?: number;
+    /**
+     * Feature flags that can be passed from consuming application.
+     * Define the feature flag name in packages/perseus-core/src/feature-flags.ts
+     */
+    flags?: Record<string, boolean>;
```

Comment:
@ivyolamit @mark-fitzgerald @SonicScrewdriver  I'm late to this, but I'd recommend that we model this as `Record<"flag1" | "flag2" | "flag3", boolean>` instead.  Otherwise, we:

* Never know from the outside what flags Perseus supports at any given time
* Within Perseus, we have no knowledge of what flags the host application may push in.  

We had the same design initially with Mafs flags and moved to be explicit instead for exactly these reasons.